arches/app/models/models.py

```diff
@@ -172,19 +175,21 @@ class Meta:
 
 
 class FormXCard(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid1) 
     form = models.ForeignKey(Form, db_column='formid')
-    parentcard = models.ForeignKey(Card, db_column='parentcardid')
+    card = models.ForeignKey(Card, db_column='cardid')
 
     class Meta:
         managed = True
         db_table = 'forms_x_card'
-        unique_together = (('form', 'parentcard'),)
+        unique_together = (('form', 'card'),)
 
 
 class NodeGroup(models.Model):
     nodegroupid = models.UUIDField(primary_key=True, default=uuid.uuid1)  # This field type is a guess.
-    cardinality = models.TextField()
+    cardinality = models.TextField(blank=True, null=True)
```

Comment:
You’re right.  I’ll make sure the change gets implemented.  

Adam  

--  
Adam Lodge
Geospatial Systems Consultant
Farallon Geographics
415.317.6625


On Friday, March 18, 2016 at 11:45 AM, Alexei Peters wrote:

> In arches/app/models/models.py (https://github.com/archesproject/arches/pull/348#discussion_r56704211):
> > > > class NodeGroup(models.Model): > nodegroupid = models.UUIDField(primary_key=True, default=uuid.uuid1) # This field type is a guess. > - cardinality = models.TextField() > + cardinality = models.TextField(blank=True, null=True)  
> I think every nodegroup needs to specify a cardinality, maybe we can specify a default value as well
>  
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub (https://github.com/archesproject/arches/pull/348/files/a54a04856816cd3909dbe3920db392ff5bbf7ccb#r56704211)
>  
>  
>  
>  
>